src/main/java/frc/robot/commands/common/arm/ArmCommand.java

```diff
@@ -22,6 +23,8 @@ public ArmCommand(Arm arm, DoubleSupplier speed) {
   @Override
   public void initialize() {
     targetPosition = arm.getAngle();
+    targetFinalPosition = 90;
```

Comment:
This isn't ideal, hard-coding the target position to 90 won't let us move the arm/climber to other positions.
Take a look at https://github.com/DevilBotz2876/Reefscape2025/blob/main/src/main/java/frc/robot/commands/common/arm/ArmToPosition.java
It takes an argument for target position.  I think that command could be used instead of this one.